doc/ale.txt

```diff
@@ -1181,6 +1181,43 @@ b:ale_loclist_msg_format                             *b:ale_loclist_msg_format*
 
   The strings for configuring `%severity%` are also used for this option.
 
+
+g:ale_lsp_show_message_format                           *g:ale_lsp_show_message_format*
+
+  Type: |String|
+  Default: `'%severity%:%linter%: %s'`
+
+  This variable defines the format that messages received from an LSP will
+  have. The following sequences of characters will be replaced.
+
+    `%s`           - replaced with the message text
+    `%linter%`     - replaced with the name of the linter
+    `%severity%`   - replaced with the severity of the message
+
+  To configure the strings used on the `%severity%` field, see |g:ale_echo_msg_format|.
+
+  The echo message format can also be configured separately for each buffer,
```

Comment:
This format is not actually configurable per buffer (I kept the text on purpose). Not sure if it even makes sense, this is not a channel for the server to flag anything specific regarding a file. I'd suggest having a dict to configure the format per server or file type but that looks like too much hassle for too little benefit really.